app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<% provide(:title, @user.first_name) %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <h1>
+        <%= gravatar_for @user %>
+        <%= @user.first_name %>
+      </h1>
+    </section>
+  </aside>
+</div>
+<%= @user.first_name %>
+<%= @user.last_name %>
+<%= @user.email %>
+<%= @user.username %>
+<%= @user.password %>
+<%= @user.password_confirmation %>
+<%= @user.height %>
+<%= @user.weight %>
+<%= @user.activities %>
+<%= @user.created_at %>
+<%= @user.updated_at %>
```

Comment:
Are these form fields?
If is, dont they need to be wrapped in `<%= form_user do %>` where `form_user` is a helper. ?